xml/depl_nodes.xml

```diff
@@ -2467,7 +2457,7 @@ getent group postgres | cut -d: -f3</screen>
     </listitem>
    </varlistentry>
    <varlistentry>
-    <term><guimenu>User Name/Password for Accessing NetApp</guimenu>
```

Comment:
In the Web UI: 'Username for Accessing NetApp' and 'Password for Accessing NetApp'